cmd/edge-orchestration/capi/main.go

```diff
@@ -325,6 +327,7 @@ func (cHandler customHandler) GetKey(id string) ([]byte, error) {
 	return []byte(key), nil
 }
 
+// SetHandler sets handler
 //export SetHandler
```

Comment:
How about removing this line?

Reply:
We got a [warnings](https://goreportcard.com/report/github.com/lf-edge/edge-home-orchestration-go#golint) from the Go Report Card